src/struct_info.json

```diff
@@ -458,7 +458,8 @@
             "S_IWUGO",
             "S_IRUGO",
             "S_IRWXUGO",
-            "S_IXUGO"
+            "S_IXUGO",
+            "PATH_MAX"
```

Comment:
Oh, I assumed we were already exporting this defined. 

Can you make a new block for `limits.h` (where MAX_PATH is defined).

You will also need to run `./test/runner other.test_gen_struct_info --rebase` to update the test expectation.

Reply:
@sbc100  Working on this. Since you mentioned one test here, I was curious enough to know what are the exact set of test commands one has to check locally before submitting the code patch and send the pull request here.